Makefile

```diff
@@ -149,21 +150,22 @@ build-runner-image:
 	docker build -t $(RUNNER_IMAGE_NAME) $(DOCKER_BUILD_EXTRA_ARGS) .
 
 run-command-%:
-	docker run --rm -t --name $(DOCKER_DEV_CONTAINER_NAME) --network host \
-	           $(DOCKER_RUN_MOUNT_OPTIONS) $(DOCKER_ENV) $(RUNNER_IMAGE_NAME) \
-	           make $* EXTRA_ARGS=$(EXTRA_ARGS)
+	docker run --rm -t --name $(DOCKER_DEV_CONTAINER_NAME) -e S3_HOSTNAME=localhost --network host \
+	    $(DOCKER_RUN_MOUNT_OPTIONS) $(DOCKER_ENV) $(RUNNER_IMAGE_NAME) \
+	    make $* EXTRA_ARGS=$(EXTRA_ARGS)
 
 # Dev commands
 start-dev: build-runner-image localstack-start run-command-install-node-packages run-dev
 
 run-dev: run-command-build-node-all run-command-package-all run-command-tf-init \
          run-command-tf-plan-apply run-command-tf-apply replace-dev-url
 
+stop-dev:
+	docker rm -f $(DOCKER_REDOCLY_CONTAINER_NAME) $(DOCKER_DEV_CONTAINER_NAME) $(DOCKER_LOCALSTACK_CONTAINER_NAME)
 
-# Swagger UI
-swagger-ui:
```

Comment:
This is unfinished task from @mahinuzzaman , he wants to use swagger as api doc and use it like postman.

Reply:
@sakibstark11 vai and me decided we wont remove it

Reply:
@melon-ruet vai, this is now a finished task